docs/devsguide/linuxdep.rst

```diff
@@ -0,0 +1,361 @@
+.. _linuxdep:
+
+===========================
+Linux Dependencies for PyNE
+===========================
+
+Something to note before you begin the installation process is the version of Python 
+that you are using. PyNE is currently supported from Python 2 through Python 3.6 
+(NOT 3.7, which is coming in future updates).
+
+
+-------------
+PyNE Library:
+-------------
+
+PyNE is what is known as a library in Python (as with many languages, 
+libraries can be built and installed to add functionalities to the language), 
+and it is designed to assist in Nuclear Engineering projects. 
+
+
+Python
+''''''
+
+Python is a popular computer language for building applications, 
+data science, and visualization. 
+
+``To Install:``
+
+Anaconda is a comprehensive Python package 
+which provides a free installation for MacOS, Windows, and Linux. Follow the instructions 
+for your device's platform `here <https://docs.anaconda.com/anaconda/install/>`_ if you would 
+like to use Anaconda. Installing Python through Anaconda will result in many of the packages and 
+dependencies for PyNE being satisfied.
+
+If you want to download Python without the rest of Anaconda, use 
+the links in the resources below. Keep in-mind that PyNE is currently 
+not supported on Python 3.7 or later, but you can follow the instructions
+for Anaconda users in the New Developer's Guide.
+
+``Resources:``
+
+#. `Python Download (Not Anaconda) <https://www.python.org/downloads/>`_
+#. `Python Wiki <https://wiki.python.org/moin/>`_
+#. `Beginner’s Guide <https://wiki.python.org/moin/BeginnersGuide>`_
+
+
+GCC
+'''
+
+GCC stands for GNU Compiler Collection, and GNU is a Unix-Like operating system 
+that allocates resources and communicates with hardware. GNU is often used with 
+a kernel called Linux. The Compiler Collection is one element that allows for 
+PyNE to work as a Python library, even though it is built with several languages.
+
+``To Install:``
+
+To download GCC, visit `Installing GCC <https://gcc.gnu.org/install/index.html>`_
```

Comment:
This should definitely rely on a package manager.  Either `apt-get` or `conda`, but if we send folx to install GCC from source, they may never return...